internal/slacklog/generator.go

```diff
@@ -291,6 +295,9 @@ func (g *HTMLGenerator) generateAttachmentText(attachment MessageAttachment) str
 // generateFileHTML : 'text/plain' な添付ファイルをHTMLに埋め込む
 // 存在しない場合、エラーを表示する
 func (g *HTMLGenerator) generateFileHTML(file *MessageFile) string {
+	if file.Size > MAX_EMBEDDED_FILE_SIZE {
+		return `<span class="file-error">ファイルサイズが大きいため、展開しません</span>`
```

Comment:
英語でオナシャス

```suggestion
		return `<span class="file-error">file size is too big to embed. please download from above link to see</span>`
```

ただ本当ならばこのあたりのエラーはフロント側で判別できるようにしておいて、適宜日本語に翻訳できるようにデザインするのが正しいんですけど、今は英語の固定埋め込みで妥協しましょう。

Reply:
了解です